Src/BootCamp.Chapter/Program.cs

```diff
@@ -9,6 +9,30 @@ class Program
     {
         static void Main(string[] args)
         {
+            Lesson3.Demo();
+        }
+        public static int getInt(string message)
+        {
+            Console.Write(message);
+
+            return Convert.ToInt32(Console.ReadLine());
+        }
+
+        public static string getString(string message)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename method 'getString' to match pascal case naming rules, consider using 'GetString'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887979)